archeryutils/targets.py

```diff
@@ -18,20 +22,61 @@
     "Beiter_hit_miss",
     "Worcester",
     "Worcester_2_ring",
+    "Custom",
 ]
 
+# TypeAlias (annotate explicitly in py3.10+)
+#: A mapping of a target ring diameter to the score for hitting that ring.
+#: Typically ordered from smallest to largest rings but this is not necessary.
```

Comment:
```suggestion
```
This is maybe TMI?

But a small example might be nice?
Or a reference to where there is an example in the `from_face_spec()` docs?

Reply:
Linked back to `from_face_spec` in the documentation itself. For the inline comment agree TMI.